userspace/falco/app_actions/print_version.cpp

```diff
@@ -23,8 +23,25 @@ application::run_result application::print_version()
 {
 	if(m_options.print_version_info)
 	{
+		auto s = new sinsp();
```

Comment:
```suggestion
		std::unique_ptr<sinsp> s(new sinsp());
```
Ensures that the destructor gets invoked.